docs/source/en/quantization/overview.md

```diff
@@ -0,0 +1,35 @@
+<!--Copyright 2024 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+
+-->
+
+# Quantization
+
+Quantization techniques focus on representing data with less information while also trying to not lose too much accuracy. This often means converting a data type to represent the same information with fewer bits. For example, if your model weights are stored as 32-bit floating points and they're quantized to 16-bit floating points, this halves the model size which makes it easier to store and reduces memory-usage. Lower precision can also speedup inference because it takes less time to perform calculations with fewer bits.
+
+<Tip>
+
+Interested in adding a new quantization method to Transformers? Read the [`DiffusersQuantizer`](../conceptual/contribution) guide to learn how!
```

Comment:
Is this supposed to link to the Transformers (https://huggingface.co/docs/transformers/main/en/quantization/contribute) doc or the Diffusers [contribution](https://huggingface.co/docs/diffusers/main/en/conceptual/contribution) doc?

Reply:
Contribution docs.

Reply:
Ah ok, would it make more sense to link to the quantization contribution doc in Transformers then?

Reply:
Ah actually yes! Could you maybe provide a suggestion?

Reply:
> Refer to the [Contribute new quantization method](https://huggingface.co/docs/transformers/main/en/quantization/contribute) guide to learn more about adding a new quantization method.

Reply:
Done in acdeb254b.